src/platform/Darwin/DnssdImpl.h

```diff
@@ -27,9 +27,15 @@
 #include <string>
 #include <vector>
 
+constexpr char kLocalDot[] = "local.";
```

Comment:
Oh, and these non-namespaced things should not be in the header... Should be inside a namespace, and probably should be inline constexpr.

Reply:
moved them back to the .cpp file under anonymous namespace as it was